src/DataFixtures/Course.php

```diff
@@ -15,37 +15,197 @@
     {
-        $summary = new Summary('summary', 'note');
-        $manager->persist($summary);
 
-        $image = new Image(
-            'http://kitlab.pef.czu.cz/1819zs/ete89e/02/static/images/classroom-2093743_1920.jpg',
-            'Java - Od začátečníka k expertovi - Učebna'
-        );
+        $imageArray = array(new Image('http://kitlab.pef.czu.cz/1819zs/ete89e/02/static/images/classroom-2093743_1920.jpg', 'Java - Od začátečníka k expertovi - Učebna'),
+            new Image('http://kitlab.pef.czu.cz/1819zs/ete89e/02/static/images/classroom-2093743_1920.jpg', 'Java - Od začátečníka k expertovi - Učebna'),
+            new Image('http://kitlab.pef.czu.cz/1819zs/ete89e/02/static/images/classroom-2093743_1920.jpg', 'Java - Od začátečníka k expertovi - Učebna'),
+            new Image('http://kitlab.pef.czu.cz/1819zs/ete89e/02/static/images/classroom-2093743_1920.jpg', 'Java - Od začátečníka k expertovi - Učebna'));
 
-        $manager->persist($image);
+        $tagArray = array(new Tag('front-end'),
+            new Tag('back-end'),
+            new Tag('front-end'),
+            new Tag('back-end'));
 
-        $address = new Address('U Vody', 'Prague');
-        $manager->persist($address);
+        $addressArray = array(new Address('U Vody', 'Prague')
+        ,new Address('Falesna 123', 'Brno')
+        ,new Address('U Pergamnky 69', 'Prague')
+        ,new Address('Dělnická 11', 'Prague'));
 
-        $tag = new Tag('back-end');
-        $manager->persist($tag);
-        $this->addReference(self::TAG_REFERENCE, $tag);
+        $summaryArray = array(new Summary('Pro začátečníky', '')
+        ,new Summary('Pro Mírně pokročilé', '')
+        ,new Summary('Pro středně pokročilé', '')
+        ,new Summary('Pro pokročilé', ''));
 
-        $course = new CourseEntity('title', 'desct', 1000, 20, 'content', $summary);
-        $course->addAddress($address);
-        $course->addImage($image);
-        $course->addTag($tag);
+        $titleArray = array("Java - Mírně pokročilý", "Java - Středně pokročilý", "Python - Mírně pokročilý", "Pyton - Středně pokročilý");
 
-        $rating = new Rating('Good', 3, $course);
-        $course->addRating($rating);
+        $priceArray = array(1000, 2000, 3000, 4000);
 
-        $manager->persist($course);
+        $durationArray = array(10, 20, 30, 40);
 
-        $manager->flush();
+        $ratingArray = array("Good", "Awesome", "Well done", "Great");
+        $ratingValueArray = array(1, 2, 3, 4);
+
+        $contentArray = array("Lekce budou probíhat se zaměřením na odlišnoti jazyka", "Lekce probíhají v interaktivních učebnách", "Je potřeba si vzít s sebou NTB", "Stolní počítače jsou přítomny na místě");
+
+        for($i = 0; $i<4; $i++) {
+            $manager->persist($summaryArray[$i]);
+            $manager->persist($imageArray[$i]);
+            $manager->persist($addressArray[$i]);
+            $manager->persist($tagArray[$i]);
+
+            switch ($i) {
```

Comment:
jak jsem psal:
```
$this->addReference(sprintf("%s_%s", self::TAG_REFERENCE, $i), $tagArray[$i]);
```
vytvori reference `tag-user_1`, `tag-user_2` apod. to nestačí?

Reply:
Kdyz se to pokousim retezit dynamicky, tak mi to nejde. Proto jsem to vyresil takhle.